paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/AddPaymentMethodOptions.kt

```diff
@@ -0,0 +1,54 @@
+package com.stripe.android.link.ui.wallet
+
+import com.stripe.android.link.LinkConfiguration
+import com.stripe.android.link.LinkLaunchMode
+import com.stripe.android.link.LinkPaymentMethodFilter
+import com.stripe.android.link.model.LinkAccount
+import com.stripe.android.link.model.supportedPaymentMethodTypes
+import com.stripe.android.model.ConsumerPaymentDetails
+import dagger.assisted.Assisted
+import dagger.assisted.AssistedFactory
+import dagger.assisted.AssistedInject
+
+internal class AddPaymentMethodOptions @AssistedInject constructor(
```

Comment:
👀 Consolidated logic to this class, used in both `LinkActivityViewModel` and `WalletViewModel`.